artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/components/consumers.js

```diff
@@ -106,7 +106,8 @@ var Artemis;
               {name: "Messages Acknowledged", visible: false},
               {name: "Messages Acknowledged awaiting Commit", visible: false},
               {name: "Last Delivered Time", visible: false},
-              {name: "Last Acknowledged Time", visible: false}
+              {name: "Last Acknowledged Time", visible: false},
+              {name "status", visible: false}
```

Comment:
Missing a colon here. It should be `{name: "status", visible: false}`

Reply:
thanks a lot.. that's why it was not working.